google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -717,15 +695,34 @@ def split_pdf(self, pdf_path: str, output_path: str) -> List[str]:
         return output_files
 
     def convert_document_to_annotate_file_response(self) -> AnnotateFileResponse:
-        r"""Convert OCR data from Document.proto to AnnotateFileResponse.proto for Vision API.
+        r"""Convert OCR data from `Document.proto` to `AnnotateFileResponse.proto` for Vision API.
 
         Args:
             None.
         Returns:
             AnnotateFileResponse:
-                Proto with TextAnnotations.
+                Proto with `TextAnnotations`.
         """
-        return _convert_to_vision_annotate_file_response(self.text, self.pages)
+        responses: List[AnnotateImageResponse] = []
+
+        for shard in self.shards:
+            for docai_page in shard.pages:
+                page_vision_annotation = _convert_document_page(
```

Comment:
Can we take this opportunity to import module instead of functions?  In particular this would become `vision_helpers._convert_document_page`, making its use cases more obvious.

Reply:
I'm good with that. This was just moving the code that was already present.